modules/ROOT/pages/_partials/sync/server-files.adoc

```diff
@@ -0,0 +1,54 @@
+= File Upload Slient
+
+{sync-server} provides support for uploading binary data along with the GraphQL queries.
+Implementation is relying on upstream `Apollo NodeJS` server capabilities
+
+== Introduction
+
+The upload functionality using the GraphQL multipart form requests specification.
+The File Upload needs to be implemented on both server and client:
+
+1) On the client HTML FileList objects are mapped into a mutation and sent to the server in a multipart request.
+
+2) On the server: The multipart request is handled. The server processes it and provides an upload argument to a resolver. 
+In the resolver function, the upload promise resolves an object.
```

Comment:
would 1 and 2 be better in the intro?

Reply:
oh, it's already in the client side intro, doh!
so, so we need it in both places?

Reply:
It is essential for the reader to know the premise, but not sure how to do that properly with the boundaries of server/client. Duplication is small so maybe we can close eye? Any suggestions welcome

Reply:
As part of sync docs review I'll try to merge, fine for now